save-orchestrator/build.gradle.kts

```diff
@@ -92,7 +92,3 @@ kotlin.sourceSets.getByName("main") {
 tasks.withType<KotlinCompile>().forEach {
     it.dependsOn(generateVersionFileTaskProvider)
 }
-
-tasks.withType<Test> {
```

Comment:
Why?

Reply:
I thought that it pollutes logs too much, on CI it's hard to find which test has failed among all those log lines. Do you find it useful? I usually navigate to HTML reports anyway, but that may be just me.

Reply:
Usually I used it in course of development, however, yes, we've a got a lot of extra logs, so I can enable it manually if I need